pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -31,6 +31,10 @@ let
       mv System/* .
       rmdir System
 
+      pushd lib
+      ln -s -L /usr/lib/libcups*.dylib .
```

Comment:
The problem is that our SDK is too old to build current `cups`. We could probably identify an older version that works, but I worry about causing confusion with `nixpkgs.cups` that is the current release.